### ISSUE TITLE
Filter bar is not accessible using keyboard navigation

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/language-server-protocol-inspector#18]
https://github.com/Microsoft/vscode/issues/52866

**Repo**
1.  Try tabbing through LSP inspector page
1. Try reaching to `filter-bar` entries (such as `Workspace` or `Text`)

**Bug**
I was not able to reach the entries in `filter-bar` using keyboard navigation. I believe these elements should be `<button>` instead of `<span>` or set the proper aria attributes
